liquid-platform/liquid-platform.cabal

```diff
@@ -25,14 +25,14 @@ flag legacy
 
 executable liquid
   main-is:            src/Liquid.hs
-  build-depends:      liquid-base       >= 0.8.10.1
-                    , liquid-containers >= 0.8.10.1
+  build-depends:      liquid-base       >= 4.14.0.0 && < 5
+                    , liquid-containers >= 0.6.0.0  && < 7
                     , liquid-prelude    >= 0.8.10.1
-                    , liquid-vector     >= 0.8.10.1
-                    , liquid-bytestring >= 0.8.10.1
+                    , liquid-vector     >= 0.12.1.2 && < 0.13
+                    , liquid-bytestring >= 0.10.0.0 && < 0.10.11.0
                     , liquidhaskell     >= 0.8.10.1
-                    , process           >= 1.6.0.0
-                    , cmdargs           >= 0.10
+                    , process           >= 1.6.0.0 && < 1.7.0.0
+                    , cmdargs           >= 0.10    && < 0.12
   default-language:   Haskell98
   ghc-options:        -W -O2 -threaded -fdefer-typed-holes
```

Comment:
`-fdefer-typed-holes` does not belong in a release package.

Reply:
Yes, it's what the old executable used to provide, and this got copy/pasted as a result. I'll remove it.